docker/Dockerfile

```diff
@@ -61,7 +61,31 @@ RUN mkdir /usr/TASO
 
 RUN cd /usr && wget https://www2.graphviz.org/Packages/stable/portable_source/graphviz-2.44.0.tar.gz && tar -xvf graphviz-2.44.0.tar.gz && cd graphviz-2.44.0 && ./configure && make && make install
 
+# gdb parts
 #RUN apt-get install -y texinfo
 #RUN cd /usr && wget "http://ftp.gnu.org/gnu/gdb/gdb-9.2.tar.gz" && tar -xvzf gdb-9.2.tar.gz && cd gdb-9.2 && mkdir build && cd build && ../configure && make & make install
 
+# or-tools parts
+RUN apt-get update && apt-get -y install git wget pkg-config build-essential cmake autoconf libtool zlib1g-dev lsb-release libreadline-dev && apt-get -y install swig
+RUN git clone https://github.com/google/or-tools /usr/or-tools
+
+# Install SCIP (MILP solver)
+#Get archive https://scip.zib.de/download.php?fname=scipoptsuite-7.0.1.tgz (might need to download to local computer then use scp)
```

Comment:
This is for record here, so I can run these to install SCIP whenever I create a new docker container. SCIP doesn't have a link directly for download, I need to fill in a form and it will start download, so I wasn't able to wget here directly (thus not making it docker commands). I can figure out how to get docker on this later